Cargo.toml

```diff
@@ -11,5 +11,5 @@ readme = "README.md"
 
 [dependencies]
 crc16 = "0.4"
-rand = "0.5"
-redis = "0.10"
+rand = "0.7"
```

Comment:
Maybe enable dependabot for bumping dependencies?

https://github.com/marketplace/dependabot-preview

Reply:
Thank you for your information.
I didn't know that.
I will think about whether to do that.